src/internal/common/maps/keyset_test.go

```diff
@@ -0,0 +1,78 @@
+package maps
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/tester"
+)
+
+type KeySetTestSuite struct {
+	tester.Suite
+}
+
+func TestKeySetTestSuite(t *testing.T) {
+	suite.Run(t, &KeySetTestSuite{Suite: tester.NewUnitSuite(t)})
+}
+
+func (suite *KeySetTestSuite) Test_HasKey() {
```

Comment:
Nit: no underscore needed when naming the function under test directly.  Underscores are for methods, variables, and extensions.  eg:
```go
func TestStruct()
func TestStruct_Method()
func TestFunc()
func TestStruct_hasSpecificProperty()
func TestStruct_Method_doesSomethingSpecific()
func TestFunc_doesSomethingSpecific()
```
```suggestion
func (suite *KeySetTestSuite) TestHasKey() {
```